core/templates/pages/topic-viewer-page/topic-story-section/topic-lesson-card/topic-lesson-card.component.css

```
@@ -1,71 +1,211 @@
 .topic-lesson-card {
+  --lesson-card-header-offset: 64px;
   background: #fff;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)